cdap-app-fabric/src/main/java/co/cask/cdap/app/verification/AbstractVerifier.java

```diff
@@ -26,7 +26,7 @@
 public abstract class AbstractVerifier<T> implements Verifier<T> {
 
   protected boolean isId(final String name) {
```

Comment:
Ignore this for now, I think we need to refactor the ``Id`` class a little to share this ``CharMatcher``. Perhaps, create a jira for now with a TODO here.